exampleCourse/questions/demo/randomFakeData/server.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+import random
+
+from faker import Faker
+
+
+def formatDate(date):
+    return date.strftime("%B %d, %Y")
```

Comment:
Nit: inline this? It's only used in one place.

Reply:
Done! https://github.com/PrairieLearn/PrairieLearn/pull/7316/commits/1fe953d0abf8ddfc56b5129fa24577fa5214381a